spec.bs

```diff
@@ -572,14 +572,18 @@ scope=] |debugScope| and an optional [=debug details=] or null
 
 <div algorithm>
 To <dfn>determine if a report should be sent deterministically</dfn> given a
```

Comment:
I think we need to update the calls to this algorithm as well

Reply:
Done.